### PR TITLE
Enable security check for ubuntu 18.04, debian9 & centos7

### PR DESCRIPTION
**Pull request by smukherj1**
This has already been enabled for ubuntu 16.04

### CODE REVIEW

File: centos7/BUILD [hunk 1]
```diff
@@ -75,6 +76,13 @@ container_image(
     ],
 )
 
+# Run the security check script to generate a metadata YAML file indicating
+# whether the centos 7 has critical security vulnerability fixes.
+security_check(
+    name = "metadata",
+    image = "gcr.io/asci-toolchain/centos7-xingao-test:latest",
```
Comment (donmccasland) [inline centos7/BUILD]: eventually we're going to want these to be gcr.io/gcp-runtimes/etcetc

File: ubuntu1804/.bazelrc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google Inc. All rights reserved..
+# Copyright 2017 Google Inc. All rights reserved.
```
Comment (donmccasland) [inline ubuntu1804/.bazelrc]: should we update the year to 2019 since we're here?

File: centos7/BUILD (hunk 1 above)
Comment (smukherj1) [reply centos7/BUILD]: This needs to happen when we change the cloudbuild.yaml files.

File: ubuntu1804/.bazelrc (hunk 2 above)
Comment (smukherj1) [reply ubuntu1804/.bazelrc]: this year needs to be when the repo was open sourced, not when this file was last updated.

**Pull request merged**